src/plugProjectKonoU/khCaveResult.cpp

```diff
@@ -424,1096 +71,359 @@ blr
-lwz      r3, 0x40(r31)
-lis      r5, 0x6F6E3031@ha
-lis      r4, 0x004E6963@ha
-lwz      r12, 0(r3)
-addi     r6, r5, 0x6F6E3031@l
-addi     r5, r4, 0x004E6963@l
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-bl       setInfAlpha__Q22kh6ScreenFP7J2DPane
-li       r0, 0
-stw      r0, 0xb4(r31)
-stw      r0, 0xb8(r31)
-lwz      r3, 0x1c(r30)
-lwz      r0, 0x18(r30)
-lwz      r4, 0x14(r30)
-add      r0, r3, r0
-subf     r0, r4, r0
-stw      r0, 0xc4(r31)
-lwz      r0, 0x20(r30)
-stw      r0, 0xc8(r31)
-lwz      r3, 0x24(r30)
-lwz      r0, 0x28(r30)
-subf     r0, r3, r0
-stw      r0, 0xcc(r31)
-lbz      r0, 0x30(r30)
-cmplwi   r0, 0
-beq      lbl_803F8D0C
-lwz      r3, 0x40(r31)
-lis      r5, 0x6D656E75@ha
-lis      r4, 0x4E66695F@ha
-lwz      r12, 0(r3)
-addi     r6, r5, 0x6D656E75@l
-addi     r5, r4, 0x4E66695F@l
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-li       r0, 0
-lis      r5, 0x6D656E75@ha
-stb      r0, 0xb0(r3)
-lis      r4, 0x4E636F5F@ha
-addi     r6, r5, 0x6D656E75@l
-lwz      r3, 0x40(r31)
-addi     r5, r4, 0x4E636F5F@l
-lwz      r12, 0(r3)
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-li       r0, 1
-lis      r5, 0x6D703031@ha
-lis      r4, 0x0050636F@ha
-stb      r0, 0xb0(r3)
-addi     r26, r5, 0x6D703031@l
-addi     r27, r4, 0x0050636F@l
-b        lbl_803F8D70
+	DispCaveResult* disp = static_cast<DispCaveResult*>(getDispMember());
 
-lbl_803F8D0C:
-lwz      r3, 0x40(r31)
-lis      r5, 0x6D656E75@ha
-lis      r4, 0x4E66695F@ha
-lwz      r12, 0(r3)
-addi     r6, r5, 0x6D656E75@l
-addi     r5, r4, 0x4E66695F@l
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-li       r0, 1
-lis      r5, 0x6D656E75@ha
-stb      r0, 0xb0(r3)
-lis      r4, 0x4E636F5F@ha
-addi     r6, r5, 0x6D656E75@l
-lwz      r3, 0x40(r31)
-addi     r5, r4, 0x4E636F5F@l
-lwz      r12, 0(r3)
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-li       r0, 0
-lis      r4, 0x696E3031@ha
-stb      r0, 0xb0(r3)
-addi     r26, r4, 0x696E3031@l
-li       r27, 0x5066
+	m_screenMain = new P2DScreen::Mgr_tuning;
+	m_screenMain->set("result_doukutu.blo", 0x1040000, arc);
 
-lbl_803F8D70:
-lbz      r0, 0x31(r30)
-cmplwi   r0, 0
-bne      lbl_803F8D8C
-lwz      r3, 0x20(r30)
-lwz      r0, 0x1c(r30)
-cmplw    r3, r0
-beq      lbl_803F8DE8
+	m_screenDropItem = new P2DScreen::Mgr_tuning;
+	m_screenDropItem->set("result_doukutu_drop_item.blo", 0x1040000, arc);
 
-lbl_803F8D8C:
-lwz      r3, 0x40(r31)
-lis      r5, 0x6E6F726D@ha
-lis      r4, 0x50616E61@ha
-lwz      r12, 0(r3)
-addi     r6, r5, 0x6E6F726D@l
-addi     r5, r4, 0x50616E61@l
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-li       r0, 1
-lis      r5, 0x636F6D70@ha
-stb      r0, 0xb0(r3)
-lis      r4, 0x50616E61@ha
-addi     r6, r5, 0x636F6D70@l
-lwz      r3, 0x40(r31)
-addi     r5, r4, 0x50616E61@l
-lwz      r12, 0(r3)
-lwz      r12, 0x3c(r12)
-mtctr    r12
-bctrl
-li       r0, 0
-stb      r0, 0xb0(r3)
-b        lbl_803F8E40
+	void* file      = JKRFileLoader::getGlbResource("result_doukutu.bck", arc);
```

Comment:
Could this pattern be abstracted into a define? perhaps `LOAD_TO_TYPE_FROM("asdfasdfas", J2DAnmTransform, arc);`